### PR TITLE
Use custom RabbitMQP setup

### PR DESCRIPTION
**Pull request by LucWollants**
### Changed
- Used custom RabbitMQ image with preconfigured exchange and queues equal to Vagrant setup

### CODE REVIEW

File: docker.md [hunk 1]
```diff
@@ -30,19 +30,7 @@ Copy the `public-auth0.pem` from https://github.com/cultuurnet/udb3-vagrant/blob
...
-  port: 5672
-  vhost: /
-  user: guest
-  password: guest
-```
-
-To read messages from [udb3-backend](https://github.com/cultuurnet/udb3-backend):
-- Create an exchange `udb3.x.domain-events` in your RabbitMQ provider
-- Communication between Docker containers is handled through `host.docker.internal`
+Login to the management console on http://host.docker.internal:15672/ with username `vagrant` and password `vagrant`
```
Comment (JonasVHG) [inline docker.md:33]:
```suggestion
Login to the management console on http://host.docker.internal:15672/ with username `guest` and password `guest`
```

**Review by JonasVHG — CHANGES_REQUESTED** [review]
There seems to be a small typo in the login data of the management console.

File: docker.md (hunk 1 above)
Comment (LucWollants) [reply docker.md:33]: No, the [custom RabbitMQ image](https://github.com/cultuurnet/udb3-backend/pull/1288/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R50) used in `udb3-backend` has everything set up like the Vagrant box. So maybe it's a bit confusing but vagrant is used for username, password, and host 😉

Comment (JonasVHG) [reply docker.md:33]: My mistake 🫣, I only deleted my search-containers not my rabbitMq container.

**Review by JonasVHG — APPROVED** [review]

**Pull request merged**